src/typeguards/UBA.ts

```diff
@@ -0,0 +1,17 @@
+import { FillWithBlock, RefundRequestWithBlock, UbaFlow, UbaInflow, UbaOutflow } from "../interfaces";
+
+export const isUbaInflow = (flow: UbaFlow): flow is UbaInflow => {
+  return (flow as UbaInflow).quoteTimestamp !== undefined;
+};
+
+export const isUbaOutflow = (flow: UbaFlow): flow is UbaOutflow => {
+  return !isUbaInflow(flow) && (outflowIsFill(flow) || outflowIsRefund(flow));
+};
+
+export const outflowIsFill = (outflow: UbaOutflow): outflow is FillWithBlock => {
+  return (outflow as FillWithBlock).isSlowRelay !== undefined;
```

Comment:
This should be `updatableRelayData.isSlowRelay`, have you tried rebasing off my across-2.5 branch?